main.tf

```diff
@@ -0,0 +1,74 @@
+data "aws_caller_identity" "current" {
+}
+
+locals {
+  account_id = data.aws_caller_identity.current.account_id
+
+  # A map with the region is used for clarity, but only the account ID is needed
+  elb_account_ids = [
+    for key in keys(var.elb_logging_regions) : "arn:aws:iam::${var.elb_logging_regions[key]}:root"
+  ]
+
+}
+
+resource "aws_s3_bucket" "this" {
+  bucket = "${local.account_id}-${var.region}-${var.bucket_suffix}"
```

Comment:
does it make sense to use the tagging module to do this? 
```
module "tags" {
  source  = "rhythmictech/tags/terraform"
  version = "0.0.2"

  names = [local.account_id, var.region, var.bucket_suffix]
}
```

Reply:
That's a nifty idea, but the bucket names are immutable, so if a change in the tag module happened to do something silly, it could cause tf to try to recreate the bucket. Which is unnecessarily scary, methinks.